README.md

```diff
@@ -15,6 +15,14 @@ flamegraph generation library!
 
 Windows is getting [dtrace support](https://techcommunity.microsoft.com/t5/Windows-Kernel-Internals/DTrace-on-Windows/ba-p/362902), so if you try this out please let us know how it goes :D
 
+**Note**: If you're using lld on on Linux, you must use the `--no-rosegment` flag. Otherwise perf will not be able to generate accurate stack traces. For example
```

Comment:
```suggestion
**Note**: If you're using lld on Linux, you must use the `--no-rosegment` flag. Otherwise perf will not be able to generate accurate stack traces. For example
```

Maybe link to the Chromium issue here, as well?

Reply:
Done